x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -230,9 +234,15 @@ export const ActionForm = ({
       .list()
       /**
        * TODO: Remove when cases connector is available across Kibana. Issue: https://github.com/elastic/kibana/issues/82502.
+       * TODO: Need to decide about ServiceNow SIR connector.
```

Comment:
decide what?

Reply:
The alerting team needs to decide if they gonna support ServiceNow SIR connector for alerts.